docs/sql_reference/src/asciidoc/_chapters/sql_statements.adoc

```diff
@@ -2235,13 +2235,19 @@ session.
 
 ```
 CREATE [VOLATILE] TABLE IF NOT EXISTS table
-    { table-spec | like-spec }
+    table-spec
     [SALT USING num PARTITIONS [ON (column[, column]...)]]
     [STORE BY {PRIMARY KEY | (key-column-list)}]
-    [HBASEOPTIONS (hbase-options-list)]
+    [HBASE_OPTIONS (hbase-options-list)]
     [LOAD IF EXISTS | NO LOAD]
     [AS select-query]
 
+CREATE [VOLATILE] TABLE IF NOT EXISTS table
```

Comment:
Not your issue but happened to notice it.  Isn't the IF NOT EXISTS clause optional?